### DISCUSSION TITLE
Weird UI layouts

### DISCUSSION THREAD

**Discussion by afbagwell** [vladmandic/sdnext#2123] [General]
I'm using bethecloud/storj_theme on Firefox 117.0 (64-bit), and this might be completely confined to it, but I'm having some weird sizing issues of UI elements since the last gradio update. The example below is the system paths tab where everything is getting crowded to the left and wrapping the text.

<img width="1908" alt="Screenshot 2023-09-01 094943" src="https://github.com/vladmandic/automatic/assets/1495201/fd031191-d929-4a82-82b6-7fa15e4d4d1b">

There are similar issues with other tabs extending fields all the way over to the right edge of the screen.

Anyone else seeing stuff like this happen? Are other themes similarly affected?

**Comment by vladmandic** [comment]
fixed in dev, will be published to main soon.

**Comment by PresTrembleyIIIEsq** [comment]
Another data point: I see similar with the default theme. Lots of buttons/sliders work, but most importantly, some buttons like the Generate button don't do anything now.

Nothing else has changed on my setup since everything last worked properly; no ad blockers or other extensions installed. Using Firefox on Linux (Manjaro).

![Screenshot_20230901_115116](https://github.com/vladmandic/automatic/assets/140981996/949db52e-6f49-41ea-961d-4e0a303dfae1)
